### DISCUSSION TITLE
Where are Autopilot devices located in the folder structure when backing them up?

### DISCUSSION THREAD

**Discussion by gmenziesint** [almenscorner/IntuneCD#152] [Q&A] [answered]
I'm unable to find the location of Autopilot devices in the folder structure even though I have --autopilot=true set in the arguments for startbackup.

![image](https://github.com/almenscorner/IntuneCD/assets/138468674/a64bdfe5-bd57-4316-8cf0-5c8701cc4cb2)

![image](https://github.com/almenscorner/IntuneCD/assets/138468674/6759c93d-b9f7-4da9-bac5-4cd506990a96)

Thanks!

**Comment by almenscorner** [comment, marked as answer]
Try set it to `True` instead of `true`. I really need to change that to a `store_true` argument :)

**Reply by gmenziesint** [reply]
That worked.. oddly enough I could swear I tried that.. Thanks!
